assets/javascripts/discourse/templates/preferences/chat.hbs

```diff
@@ -25,18 +25,6 @@
   </span>
 </div>
 
-<div class="control-group chat-setting">
-  <label class="controls">
-    {{input
-      id="user_chat_isolated"
-      type="checkbox"
-      checked=model.user_option.chat_isolated
-    }}
-    {{i18n "chat.isolate.title"}}
-  </label>
-  <span class="control-instructions">{{i18n "chat.isolate.description"}}</span>
```

Comment:
You might want to remove those strings from the locale file as well.

Reply:
done